### DISCUSSION TITLE
Get Rid of Fall Damage

### DISCUSSION THREAD

**Discussion by gerakis** [FakeFishGames/Barotrauma#7612] [Ideas and suggestions]
Fall damage adds nothing to the game except annoyance. This isn't a platformer. I don't think anyone is enjoying this as a game mechanic. It almost invariably means "I missed the ladder somehow" instead of something meaningful happening. There goes another morphine down the drain.

I don't really care if its realistic, but if its important to you Europa's gravity is about 13% of earths gravity. It seems reasonable that falling wouldn't hurt there, unless you fell hundreds of feet or something.

Devs, please take out fall damage.

**Comment by evilfactory** [comment]
Fall damage is so low that it isn't noticeable, even when you jump from a very high place, you take minimal amounts of damage, even if you get damaged by fall damage 10 times in a row, a single opium will fix all damage.

https://user-images.githubusercontent.com/36804725/144757013-aee77f94-0c13-4fe6-89d6-de48f369f83c.mp4

**Comment by MaaiG** [comment]
Anyway, it’s strange that falling from different heights leads to the same injury. And get off with a broken leg (Left leg!)
And it’s obvious that the acceleration is increasing!

https://user-images.githubusercontent.com/93348499/144759743-160e312d-c2dc-40ea-901d-27b691bece37.mp4

Even when the character punched through the floor!
**From the submarine manual:** _"When you run out of railgun shells. Load the captured pirate! Or anyone else..."_

https://user-images.githubusercontent.com/93348499/144759751-3ee0712a-7fbe-46de-b78d-e62f35c582bd.mp4

**Comment by gerakis** [comment]
Are you enjoying the damage though? If so can you explain why?

**Comment by MaaiG** [comment]
I’m not Yes or No, I’m for rethinking. But I’m definitely against those betting "No" and refusing to write - "Why"!
Falling damage from what height are we talking? The character falling from the second floor does not suffer damage.
If we’re talking about cleaning up the damage at all, I’m probably against it, too. You can twisted your ankle when standing in one place. There may be a need to reconsider the type of damage, much less the damage at higher speeds. As I showed in the video, the character breaks one leg by puncturing two bulkheads. Besides, IMHO, the moon European is still a convention that justifies the presence of alien structures and unseen monsters, but far from real physics. There has to be reasonable compromise.
